app/src/main/java/com/czech/chronos/ui/screens/HomeScreen.kt

```diff
@@ -29,26 +26,67 @@ import androidx.hilt.navigation.compose.hiltViewModel
 import com.czech.chronos.R
 import com.czech.chronos.ui.components.HomeFeatures
 import com.czech.chronos.ui.viewModels.HomeViewModel
+import com.czech.chronos.utils.DateUtil
 import com.czech.chronos.utils.Fonts
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.withContext
+import java.time.LocalDateTime
+import java.time.ZoneId
+import java.time.ZonedDateTime
+import java.time.format.DateTimeFormatter
 
+@OptIn(ExperimentalMaterial3Api::class)
 @RequiresApi(Build.VERSION_CODES.O)
 @Composable
 fun HomeScreen(
```

Comment:
One nice way of creating screen composables is the `Screen`, `Content`, ``Preview` approach. 
You have a public `FeatureScreen` which accepts a viewmodel.
A private `Content` composable which is a pure composable with no dependency on Android.
a private `Preview` composable which previews `Content` 

From `Screen` composable, you can call `Content` with view states. 
From `Preview` composable, you can show a preview of `Content` with any view state you create.